common/config/last9.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"errors"
+
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	l9OtlpEndpointKey   = "LAST9_OTLP_ENDPOINT"
+	l9OtlpAuthHeaderKey = "LAST9_OTLP_BASIC_AUTH_HEADER"
+)
+
+type MyDest struct{}
```

Comment:
```suggestion
type Last9 struct{}
```

also for the functions on the object in this file